torch/csrc/deploy/interpreter/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ add_custom_command(
    OUTPUT ${FROZEN_FILES}
    WORKING_DIRECTORY ${INTERPRETER_DIR}
    COMMAND mkdir -p ${FROZEN_DIR}
-   COMMAND ${PYTHON_BIN} freeze.py ${PYTHON_STDLIB_DIR} ${TYPING_PKG} ${PYTORCH_ROOT}/torch --oss --install_dir ${FROZEN_DIR} --verbose
+   COMMAND ${PYTHON_BIN} ../../../utils/_freeze.py ${PYTHON_STDLIB_DIR} ${TYPING_PKG} ${PYTORCH_ROOT}/torch --oss --install_dir ${FROZEN_DIR} --verbose
```

Comment:
nit: `${PYTORCH_ROOT}/utils/_freeze.py` is more robust and readable.

Reply:
Indeed, too many dots on the path makes things easy to mess up. I got the idea. Looks like the correct path using PYTORCH_ROOT should be `${PYTORCH_ROOT}/torch/utils/_freeze.py` ?

Reply:
yep!